app/util.js

```diff
@@ -296,5 +296,19 @@ module.exports = {
     } catch (err) {
         return t('notification.error.github');
     }
+  },
+
+  replaceVariables: function(variables, setting) {
+    var result = setting;
+    if (result) {
+      for (var name in variables) {
+        if (variables.hasOwnProperty(name)) {
```

Comment:
Since you're looping over each property of `variables` in the parent loop, wouldn't this always be true?

Reply:
In fact, I think if this were to match the initial functionality, it would have to be `if (result === name)`. However that would fail the replacement url for `site_url` and the like. So maybe `string.indexOf` is the better fit here.

Reply:
I was just using a common Javascript pattern (which avoids using properties inherited from any prototype object). I agree that it isn't needed here, so I'll remove it.

I'm not sure what you mean by using `String.indexOf`. Is the idea to break and recombine the string instead of doing `result.replace`?